### PR TITLE
Update resource file project settings after renaming/moving the files

### LINKED ISSUE #19132: renaming the main scene doesn't update resource
**Godot version:** 3.0.2

**OS/device including version:** fedora 28

**Issue description:**
using the provided rename function in the file manager to rename the main scene should also change the name/resource in project.godot

**Steps to reproduce:**
- create a project
- create and set a main scene
- use the file manager in godot to rename said scene
- try to run the program and get an error

### PR DESCRIPTION
**Pull request by Nallebeorn**
Fixes #19132.

### CODE REVIEW

File: editor/filesystem_dock.cpp [hunk 1]
```diff
@@ -898,6 +898,12 @@ void FileSystemDock::_update_resource_paths_after_move(const Map<String, String>
 			EditorNode::get_editor_data().set_scene_path(i, path);
 		}
 	}
+
+	String main_scene = GLOBAL_DEF("application/run/main_scene", "");
```
Comment (akien-mga) [inline editor/filesystem_dock.cpp]: You should use `GLOBAL_GET`, not `_DEF` (`DEF` is to define it).

**Comment by Nallebeorn** [comment]
As @0ttoman mentioned in #19132, there are some other resources with the same problem, so I'll extend this PR to cover all of them before it's merged.

**Comment by mhilbrunner** [comment]
Can you fix the conflicts? :)

**Comment by Nallebeorn** [comment]
Looks like GitHub's online editor inserted spaces instead of tabs, but I think I managed to fix the conflict properly locally and push it now :)

**Pull request merged**